src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -73,6 +73,7 @@ public static class ScriptConstants
         public const int AzureProxyFunctionMaxLocalRedirects = 10;
 
         public const string FeatureFlagDisableShadowCopy = "DisableShadowCopy";
+        public const string FeatureFlagDisableCSharpCompilationCache = "DisableCSharpCompilationCache";
```

Comment:
Wonder if this shouldn't just be DisableCompilationCache? Presumably in the future we could enable cache for F# as well and we'd likely want 1 switch?

Reply:
That's how I had it initially, and switched to the more specific `DisableCSharpCompilationCache` just in case we wanted the flexibility when/if others were introduced. Would be happy to change it to the more generic version if we don't think we want the flexibility. We could add the more granular option later if needed.